src/routes/integrations/storage-s3/+page.markdoc

```diff
@@ -56,6 +56,18 @@ _APP_STORAGE_S3_ACCESS_KEY=[ACCESS_KEY_ID]
 _APP_STORAGE_S3_SECRET=[SECRET_ACCESS_KEY]
 ```
 
+Starting from version 1.7.0, you can also use S3-compatible providers by configuring the endpoint:
+
+```bash
+# For path-style requests (bucket in the path)
+_APP_STORAGE_S3_ENDPOINT=[PROVIDER_ENDPOINT] # e.g., s3.amazonaws.com (without https://)
```

Comment:
Would you please update this page to use angle brackets for the placeholders?

Reply:
Done!